inst/Description.qml

```diff
@@ -32,6 +32,13 @@ Description
 		qml: 	"IntraclassCorrelation.qml"
 		func: 	"IntraclassCorrelation"
 	}
+	
+		Analysis
+	{
+		title:	qsTr("Cohen's Kappa & Fleiss' Kappa")
```

Comment:
```suggestion
		title:	qsTr("Cohen's & Fleiss' Kappa")
```
I think it should be clear from this that those are two kappa, if it were one it would probably be named Cohen-Fleiss kappa, dont you think?